task-5/index.tsx

```diff
@@ -0,0 +1,31 @@
+import React, { useEffect, useState } from 'react';
+
+function App() {
+
+    var [fuel, setFeul] = useState(0);
+    var [alertText, setAlertTxt] = useState('Processing...');

```

Comment:
```suggestion
    const [alertText, setAlertText] = useState('Processing...');
```
Typo and let's set the default alert text to something empty/null, as we don't want to show any alerts.
Also, would suggest keeping the network request pending status and response and figure out the text inline in the HTML part of the code. Just to keep logic and rendering (even text) separate. Also const instead of var.